src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
```

Comment:
_Don't use literals in expressions (HARDCODE). Instead use constants explaining this values_ - according to the checklist, please fix this